### PR TITLE
Update installation instructions to fully fix #130

### LINKED ISSUE #130: Update installation instructions
Installation instructions should be updated to be the more simple:

```
npm install -g tickbin
```

### PR DESCRIPTION
**Pull request by mitchhentges**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,5 +117,8 @@ To build the application:
 # Using node v4.2 or greater
 $ npm install
 $ npm run build
-$ ./bin/tick --help
+$ ./build/tick --help
 ```
+
+It's recommended to add an alias in your `.bashrc` (`alias tick='/PATH/TO/tickbin/build/tick'`)
```
Comment (chadfawcett) [inline README.md]: Now that we've got the package installable via npm we aren't wanting to advise on making an alias, we'll leave that up to the develop.

**Comment by chadfawcett** [comment]
Looks great.

I just made one small comment, once that is resolved I can merge this in.

File: README.md (hunk 1 above)
Comment (mitchhentges) [inline README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -121,5 +117,5 @@ To build the application:
 # Using node v4.2 or greater
 $ npm install
 $ npm run build
-$ ./bin/tick --help
-```
```
Comment (chadfawcett) [inline README.md]: Whoops, looks like you accidentally removed the ending of this code block.

Comment (mitchhentges) [inline README.md]: Ah, habit, sorry. It's `EOF`, which terminates the section of code.
Fixed to match project preference

**Comment by chadfawcett** [comment]
:100:

**Pull request merged**